eslint-plugin-kuc-v1/no-lowercase-6digits-color-code.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  create: function(context) {
+    return {
+      ClassDeclaration: function(node) {
+        const superClass = node.superClass.name;
+        if (superClass !== "KucBase") return;
+        const sourceCode = context.getSourceCode().getText();
+        const pattern = "(?<!url\\()\\#(?![a-z0-9]{6})";
```

Comment:
Ref. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Assertions#other_assertions
It will be matched only if  "```\\#(?![a-z0-9]{6})```" is not preceded by "url(".
For example, ```#fff``` is matched, ```url(#fff``` is not matched.

There is a string like url(xxx in the code and it is not the eslint check target, so I need to add ```(?<!url\\()``` to pass the check.
https://github.com/kintone-labs/kintone-ui-component/blob/e423c72d5afbabd6cb44f39a6df76dbcfd8dad11/src/mobile/radio-button/index.ts#L103

Reply:
@dong0 
Thank you for the explanation. I understand and think it is reasonable.
Please keep it.